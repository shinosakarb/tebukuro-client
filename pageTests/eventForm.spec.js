import { shallow } from 'enzyme'
import shallowToJson from 'enzyme-to-json'
import EventFormPage from '../pages/eventForm'

describe('Event page', () => {
  it('renders the event page.', () => {
    const component = shallow(<EventFormPage />)
    const tree = shallowToJson(component)

    expect(tree).toMatchSnapshot()
  })
})
