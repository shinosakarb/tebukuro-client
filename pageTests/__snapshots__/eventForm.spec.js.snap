// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event page renders the event page. 1`] = `
<div>
  <h3>
    This is the event form page!
  </h3>
</div>
`;
